pkg/cluster/internal/create/actions/installcni/cni.go

```diff
@@ -83,11 +86,39 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 		manifest = out.String()
 	}
 
+	// Add the controlplane endpoint so kindnet doesn´t have to wait for kube-proxy
+	controlPlaneEndpoint, err := ctx.Provider.GetAPIServerInternalEndpoint(ctx.Config.Name)
+	if err != nil {
+		return err
+	}
+
+	patchValue := fmt.Sprintf(`
+- op: add
+  path: /spec/template/spec/containers/0/env/-
+  value:
+    name: CONTROL_PLANE_ENDPOINT
+    value: %s
```

Comment:
this could just be string concat.

Reply:
I was trying to make it more readable :shrug:

Reply:
IMO it's harder to parse a string format in your head than recognize `"foo" + bar`

Reply:
(it's also inefficient)